### PR TITLE
Edits for JDBC_static content

### PR DESCRIPTION
**Pull request by dedemorton**
I did a quick edit of the JDBC_static plugin content and fixed the example formatting.

@acchen97 Should update the example here to include the callouts that I added to [Enriching Data with Lookups](https://www.elastic.co/guide/en/logstash/current/lookup-enrichment.html)? Wouldn't hurt, but it does duplicate some of the intro stuff. 

@guyboertje There are a couple notes for reviewers. Can you answer? thanks!

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -20,40 +20,52 @@ include::{include_path}/plugin_header.asciidoc[]
...
-structure. Define tables, columns and indexes.
+This filter works by fetching data from a remote database, caching it in a
+local, in-memory https://db.apache.org/derby/manuals/#docs_10.14[Apache Derby]
+database, and using lookups to enrich events with data cached in the local
+database. You can set up the filter to load the remote data once (for static
+data), or you can schedule remote loading to run periodically (for data that
+needs to be refreshed).
 
-Loaders
+//REVIEWERS: It's true that you can just load once, right, if you don't specify
```
Comment (acchen97) [inline docs/index.asciidoc]: that is my understanding as well. @guyboertje can confirm.

**Comment by acchen97** [comment]
@dedemorton those callouts are awesome and really help better explain what each section does. I actually think they're even more important to have those callouts here in the dedicated plugin docs. Mind adding it? :)

File: docs/index.asciidoc (hunk 1 above)
Comment (guyboertje) [reply docs/index.asciidoc]: Yes once - unless a schedule is specified (that actually works!!).

**Review by guyboertje — APPROVED** [review]
LGTM

**Pull request merged**